### ISSUE TITLE
[Spec]  Add IsLightDismissed to Popup Events

### ISSUE THREAD

**Issue by ahoefling** [xamarin/XamarinCommunityToolkit#1352]
# Add IsLightDismissed to Popup Events

Expose an `IsLightDismissed` property to the `PopupDismissedEventArgs` which will allow event listeners to understand if the popup was closed or if the user tapped in the area outside of the popup. 

*This was something that I included in the original implementation of XCT Popups that we are using internally and didn't include it as the code was migrated from Xamarin.Forms to XCT.*

# API

### Properties

| API | Description |
| ------------- | ------------- |
| IsLightDismissed | Gets or sets if the popup was dismissed via light dismiss. If true, then the user tapped outside the bounds of the popup (a light dismiss). If false, then the popup was dismissed by user action or via code. |

### Events
N/A

# Scenarios
This is a critical piece for anyone trying to build a popup library around the Xamarin Community Toolkit. We have an internal prism plugin that uses the original implementation of XCT Popups. It heavily depends on this property in the event listener mechanism. I am in the process of porting our internal library to open source and need this property exposed. Other projects that want to do similar things for other MVVM libraries will ultimately run into this problem.

# Platform Compatibility
- Target Frameworks: 
  - iOS:  ✅ 
  - Android: ✅ 
  - UWP:  ✅
 
# Backward Compatibility
There should be no need to make changes to the any dependencies as this is just exposing 1 new property.

# Difficulty : low

**Comment by ahoefling** [comment]
I spoke with @pictos in discord about this and we both agree this is a good idea. I have time this week and will be submitting a PR sometime tomorrow for this.

**Comment by pictos** [comment]
I'll mark this as `in-progress` to prevent any one to grab this issue

**Comment by ahoefling** [comment]
While working on this today, I noticed that the `Dismissed` event isn't firing for `LightDismissed` behavior. I am going to fix this in the PR as it is something that was originally missed.

Repo Steps
1. Add Dismissed event handler to a popup that has light dismiss enabled
2. Set breakpoint on event handler
3. Load app and perform light dismiss on any platform (iOS, android, uwp)
4. Perform light dismiss

Actual - The event is not triggered
Expected - The event should be triggered

When I looked at my original code for the XCT popup we trigger this event in the parent class. This was something that appears to be missed when the code was migrated from our internal code base to Xamarin.Forms and eventually to it's place here in XCT
